src/routes/staking/staking-form.tsx

```diff
@@ -203,16 +219,70 @@ export const StakingForm = ({
       </FormGroup>
       {action !== undefined && (
         <>
-          <h2>{t("How much to {{action}} in next epoch?", { action })}</h2>
-          <p>{t("Warning, spam protection exists")}</p>
-          <TokenInput
-            submitText={`${action} ${amount ? amount : ""} ${t("vegaTokens")}`}
-            perform={onSubmit}
-            amount={amount}
-            setAmount={setAmount}
-            maximum={maxDelegation}
-            currency={t("VEGA Tokens")}
-          />
+          {action === "Add" ? (
+            <>
+              <h2>{t("How much to Add in next epoch?")}</h2>
+              <p>{t("Warning, spam protection exists")}</p>
+              <TokenInput
+                submitText={`Add ${amount ? amount : ""} ${t("vegaTokens")}`}
+                perform={onSubmit}
+                amount={amount}
+                setAmount={setAmount}
+                maximum={maxDelegation}
+                currency={t("VEGA Tokens")}
+              />
+            </>
+          ) : (
+            <>
+              <h2>{t("How much to Remove?")}</h2>
+              {removeType === "now" ? (
+                <p>
+                  {t(
+                    "Removing stake mid epoch will forsake any staking rewards from that epoch"
+                  )}
+                </p>
+              ) : null}
+              <TokenInput
+                submitText={`Remove ${amount ? amount : ""} ${t(
+                  "vegaTokens"
+                )} ${
+                  removeType === "now"
```

Comment:
optional: can we enum these? Just dislike magic strings

Reply:
I think it'd also be tidier having a single translation string for this
```
t('undelegateSubmitButton', {
  amount: t('amount'),
  when: removeType === 'now' ? t('as soon as possible') : t('at the end of epoch')
)
```

Reply:
@dexturr - changed to enum
@mattrussell36 - great idea, done.